upgrade-ops-manager/gcp/pipeline.yml

```diff
@@ -29,13 +29,27 @@ resources:
     product_version: {{opsman_major_minor_version}}
     sort_by: semver
 
+- name: tool-cliaas
+  type: github-release
+  source:
+    user: pivotal-cf 
+    repository: cliaas
+    access_token: {{github_token}}
+
 - name: tool-om
   type: github-release
   source:
     user: pivotal-cf
     repository: om
     access_token: {{github_token}}
 
+- name: tool-pivnet-cli
```

Comment:
what do we need the pivnet cli for?
regardless what it's for, we should bundle it into our pcf-pipeline release on pivnet to cut down on external resources